libc/newhdrgen/yaml_to_classes.py

```diff
@@ -232,6 +232,15 @@ def main(yaml_file, h_def_file, output_dir, add_function=None):
     output_file_name = Path(h_def_file).stem
     output_file_path = Path(output_dir) / output_file_name
 
+    # with open(yaml_file, "r") as f:
```

Comment:
did you mean to leave this in?

Reply:
Just as a placeholder for now while I wait for the CMake to come out. I'm not sure if I can manually add sys to the file path here or if I will have to make an alternate path in the CMake file.

Reply:
in that case I'd say it's probably best to leave this out of this patch. Commented out code is often confusing when reading a file